app/views/layouts/_head.haml

```diff
@@ -12,7 +12,7 @@
   = render :partial => 'analytics/head_scripts', locals: { community: @current_community, user: @current_user, plan: @current_plan }
 
   :css
-    @import url(//fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700&subset=latin-ext);
+    @import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
```

Comment:
this will only download it its regular size, which is 400 wide. We need the other widths we used with Source Sans Pro: 400, 600 and 700.

Reply:
I thinks I can't do that with Montserrat Alternates Font...